blueprints/controllers/ikea_e1743/ikea_e1743.yaml

```
@@ -261,9 +315,13 @@ condition:
         {%- endif -%}
         {%- endset -%}
         {{ trigger_action not in ["","None"] }}
+#**YAR** - ADD END
       # only for zigbee2mqtt, check if the event is relative to a real state change, and not only some minor changes in the sensor attributes
```

```suggestion
      # only for zigbee2mqtt in legacy mode, check if the event is relative to a real state change, and not only some minor changes in the sensor attributes
```

Update comment matching the new condition.